packages/react-native-editor/src/index.js

```diff
@@ -136,5 +136,8 @@ const setupLocale = ( locale, extraTranslations ) => {
 	blocksRegistered = true;
 };
 
-reactNativeSetup();
-gutenbergSetup();
+export { initialHtml as initialHtmlGutenberg };
+export function setupGutenberg() {
```

Comment:
this may be nitpicking but `setupGutenberg()` and `gutenbergSetup()` are confusing to me. What do you think about renaming this method here to something like `doSetupGutenberg()` to make it clear this contains the procedure / steps to setup Gutenberg, and the other one is one of the steps (as in `xxxxSetup()`)?

Reply:
I agree, thanks for raising it. I've updated the name to be more descriptive and will merge once the CI has finished.